fetch/fetch_test.go

```diff
@@ -1365,6 +1365,23 @@ func generateTransactions(t *testing.T, count int) []*std.Tx {
 	return txs
 }
 
+// generateGenesisTransactions generates dummy genesis transactions
+func generateGenesisTransactions(t *testing.T, count int) []*gnoland.TxWithMetadata {
```

Comment:
Where do we use this?

Reply:
https://github.com/gnolang/tx-indexer/pull/111/files#diff-e10e4f3b40fd6d46955cbfceefe12b0d2d5cfbca758f9aa20cfbd0ddcdea981fR1042

Reply:
I'm blind :)